jcl/src/java.base/share/classes/java/lang/Class.java

```diff
@@ -1927,31 +1927,33 @@ String getPackageName() {
 public URL getResource(String resName) {
 	ClassLoader loader = this.getClassLoaderImpl();
 	String absoluteResName = this.toResourceName(resName);
-/*[IF Sidecar19-SE]*/
+	URL result = null;
+	/*[IF Sidecar19-SE]*/
 	Module thisModule = getModule();
-	
-	if (thisModule.isNamed() && (thisModule == System.getCallerClass().getModule())) {
+	if (useModularSearch(absoluteResName, thisModule, System.getCallerClass())) {
 		try {
-			return loader.findResource(thisModule.getName(), absoluteResName);
+			result = loader.findResource(thisModule.getName(), absoluteResName);
 		} catch (IOException e) {
 			return null;
 		}
-	} else
-/*[ENDIF] Sidecar19-SE */
+	}
+	if (null == result)
+		/*[ENDIF] Sidecar19-SE */
 	{
 		if (loader == ClassLoader.bootstrapClassLoader) {
-			return ClassLoader.getSystemResource(absoluteResName);
+			result =  ClassLoader.getSystemResource(absoluteResName);
```

Comment:
nitpick - there's some odd double spacing here and throughout the new code.  Can you address that before the merge?

Reply:
Fixed in 5 locations.